packages/react-native-codegen/src/parsers/flow/components/componentsUtils.js

```diff
@@ -34,7 +34,11 @@ function getTypeAnnotationForArray<+T>(
   typeAnnotation: $FlowFixMe,
   defaultValue: $FlowFixMe | null,
   types: TypeDeclarationMap,
-  buildSchema: (property: PropAST, types: TypeDeclarationMap) => ?NamedShape<T>,
+  buildSchema: (
+    property: PropAST,
+    types: TypeDeclarationMap,
+    language: $FlowFixMe,
```

Comment:
language is supposed to be a string I guess :) same in the below cases as well...
```suggestion
    language: string,
```

Reply:
Could you pass the `Parser` object, rather than Language?
Parser has a property that is language, but ideally, we shouldn't switch over language but leverage polymorphism thanks to the Parser object passed.